CONTRIBUTING.md

```diff
@@ -1,10 +1,10 @@
 # Fastify is an OPEN Open Source Project
 
-## What?
+## What
 
 Individuals making significant and valuable contributions are given commit-access to the project to contribute as they see fit. This project is more like an open wiki than a standard guarded open source project.
 
-### I want to be a collaborator!
+### I want to be a collaborator
```

Comment:
Why has punctuation been removed from these headings?

Reply:
Due to https://github.com/DavidAnson/markdownlint/blob/v0.22.0/doc/Rules.md#md026---trailing-punctuation-in-heading

Would you like this rule ignored or that we reword the headings slight i,e, "About" and "Become a collaborator"?